### PR TITLE
chore: fix rook-ceph resource use

### PR DESCRIPTION
**Pull request by TimJones**
Reduce Rook-Ceph cluster resource use to something more
fitting a simple set of integration tests.

Signed-off-by: Tim Jones <tim.jones@siderolabs.com>

---
This change is [Reviewable](https://reviewable.io/reviews/siderolabs/talos/5701)

### CODE REVIEW

File: hack/test/day-two/ceph-values.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    mgr:
+      limits:
+        cpu: 500m
+        memory: 512Mi
+      requests:
+        cpu: 250m
+        memory: 256Mi
+    osd:
+      limits:
+        cpu: 1000m
```
Comment (frezbo) [inline hack/test/day-two/ceph-values.yaml:12]: i would just remove the limit for cpu in tests, it just throttles the app un-necessarily

**Review by frezbo — APPROVED** [review]

**Comment by smira** [comment]
> /promote integration-qemu-day-two

that was probably too early, it allows to promote only when initial pipeline succeeds @TimJones

**Comment by TimJones** [comment]
> > /promote integration-qemu-day-two
> 
> that was probably too early, it allows to promote only when initial pipeline succeeds @TimJones

Thanks @smira, I thought the initial pipeline was green but I must have been looking at the wrong check.

File: hack/test/day-two/ceph-values.yaml (hunk 1 above)
Comment (TimJones) [reply hack/test/day-two/ceph-values.yaml:12]: If the tests pass with the limit, I'll re-test without the limit and compare.
Thanks.

File: hack/test/day-two/ceph-values.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+cephClusterSpec:
+  resources:
+    mgr:
+      limits: {}
+      requests: {}
+    osd:
+      limits: {}
+      requests: {}
```
Comment (utkuozdemir) [inline hack/test/day-two/ceph-values.yaml:8]: It probably should be like this:

```suggestion
    mgr: {}
    osd: {}
```

Comment (TimJones) [reply hack/test/day-two/ceph-values.yaml:8]: Unfortunately, the Rook operator threw some errors when the CephCluster CRD was configured that way. I'll try and dig out the logs for you.

**Pull request closed without merging**